rdkit/Chem/Pharm2D/UnitTestMatcher.py

```diff
@@ -28,10 +31,7 @@ def setUp(self):
     self.factory.SetBins([(0, 2), (2, 5), (5, 8)])
     self.factory.Init()
 
-  def test1(self):
-    """ simple tests
-
-    """
+  def test1_simple(self):
```

Comment:
I'm theoretically ok with removing the docstrings from the methods (or converting them into comments as you do below), but I'm curious why you are doing so. 
It does make the output of the tests less informative about what is going on. These docstrings are output by the Python testing code.